docs/guides/nginx/nginx-ssl.md

```diff
@@ -41,4 +44,8 @@ server {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
   }
-}
\ No newline at end of file
+}
+
+### Certbot
```

Comment:
Does this part have to be here if it isn't implemented yet? If you want to add to this PR with certbot instructions, that would be great.